payjoin/src/receive/v2/mod.rs

```diff
@@ -54,6 +55,8 @@ impl SessionContext {
             .join(&format!("/{}", directory_base))
             .map_err(|e| InternalSessionError::ParseUrl(e.into()))
     }
+
+    fn id(&self) -> ShortId { id(&self.s) }
```

Comment:
I don't think this should be here. Receiver::id() is defined and available.

Reply:
Yeah this was left over from one of the previous designs.

Reply:
Fixed